vcx/ci/Jenkinsfile

```diff
@@ -19,8 +19,8 @@ def testing() {
     stage('Testing') {
         parallel([
             'Main' : { mainUbuntu() },
-            'Android': { android() },
-            'iOS' : { ios() }
+            // 'Android': { android() },
+            // 'iOS' : { ios() }
```

Comment:
Do we need to comment this out?

Reply:
This is for testing only. Dont worry about this PR until it is posted in #utah-dev needing a Review.